dali_mqtt_daemon.py

```diff
@@ -169,6 +204,82 @@ def initialize_lamps(data_object, client):
         except DALIError as err:
             logger.error("While initializing lamp<%s>: %s", lamp, err)
 
+    for group in groups.keys():
+        logger.debug("Publishing group %d", group)
+        try:
+            logger.debug("Group %s" % group)
+            group_address = address.Group(int(group))
+            
+            actual_level = driver_object.send(gear.QueryActualLevel(group_address))
+            physical_minimum = driver_object.send(
+                gear.QueryPhysicalMinimum(group_address)
+            )
+            min_level = driver_object.send(gear.QueryMinLevel(group_address))
+            max_level = driver_object.send(gear.QueryMaxLevel(group_address))
+            device_name = 'group_%s' % group
+            lamp = device_name
```

Comment:
maybe we should rename `lamp` to `group_lamp`?

Reply:
renamed lamp variable in code